charts/latest/csi-driver-smb/templates/csi-smb-node.yaml

```diff
@@ -53,7 +53,7 @@ spec:
             - name: ADDRESS
               value: /csi/csi.sock
             - name: DRIVER_REG_SOCK_PATH
-              value: /var/lib/kubelet/plugins/smb.csi.k8s.io/csi.sock
+              value: {{ .Values.kubelet.path }}plugins/smb.csi.k8s.io/csi.sock
```

Comment:
don't need to change value here, it's inside container, let's keep `/var/lib/kubelet/`